WordPress/UITests/Tests/EditorGutenbergTests.swift

```diff
@@ -100,6 +100,8 @@ class EditorGutenbergTests: XCTestCase {
     }
 
     func testAddMediaBlocks() throws {
+        try XCTSkipIf(!XCUIDevice.isPad, "Test currently fails on iPhone in CI - Tapping on the coordinate location results in a blank screen. Skipping test on iPhone.")
```

Comment:
This problem only affects the Video and Audio blocks due to inserting media via URL. So I'm wondering if we could only skip those. Maybe we could have two different UI tests depending on the device:
1. iPhone: Test Gallery block
2. iPad: Test Gallery, Video, and Audio blocks

WDYT?

Reply:
I wouldn't recommend breaking it into two tests, especially if the blocks should behave the same on both iPhone and iPad, I also believe that this should be a temporary and not a permanent skip. 

I understand the concern around test coverage, though. So to handle this, I moved the skip after adding the image for the iPhone test so the test would still test up to add image, if the image fails to load, the test would still fail. I made the changes in https://github.com/wordpress-mobile/WordPress-iOS/pull/21582/commits/09d9f2009c42e782d2df35dd88d480bfa81a1d9e 

wdyt?

Reply:
> I wouldn't recommend breaking it into two tests, especially if the blocks should behave the same on both iPhone and iPad, I also believe that this should be a temporary and not a permanent skip.

Yeah, good point.

> I understand the concern around test coverage, though. So to handle this, I moved the skip after adding the image for the iPhone test so the test would still test up to add image, if the image fails to load, the test would still fail. I made the changes in [09d9f20](https://github.com/wordpress-mobile/WordPress-iOS/commit/09d9f2009c42e782d2df35dd88d480bfa81a1d9e)

Great idea, I'll take a look.